ci/azure-steps-windows.yml

```diff
@@ -0,0 +1,24 @@
+steps:
+- script: |
+    echo "##vso[task.prependpath]$CONDA/bin"
+  displayName: Prepare conda
```

Comment:
It doesn't look like this task did anything?

![image](https://user-images.githubusercontent.com/881019/61098548-bd5a9c00-a4a2-11e9-98f6-e508012a8864.png)


...though maybe you only get output if you run with the debug variable set

Reply:
I think perhaps it's the `$CONDA` which might not work as it's not valid `cmd.exe` syntax? It might be worth trying the pipelines syntax `$(CONDA)/bin` or `cmd.exe` syntax `%CONDA%/bin`